text/stacks/0004-jammy-jellyfish.md

```diff
@@ -0,0 +1,106 @@
+# Stacks based on Ubuntu 2022.04: Jammy Jellyfish
+
+## Summary
+
+A set of stacks based on the Ubuntu 2022.04 LTS (Jammy Jellyfish) release base
+image should be developed, released, and maintained by the Stacks subteam. Like
+the existing Bionic stacks, these new stacks should come in "full", "base", and
+"tiny" variants with similar, if not identical, sets of packages pre-installed.
```

Comment:
Will we be shipping two sets of each stack like we do for bionic? One without CNB metadata, and one with it (stacks with the `-cnb` suffix)?

Reply:
No. I'd like to see us drop the set of images that isn't a valid CNB stack. We should just ship valid CNB stacks. If users want to build on top of our stack, they can just do that directly. We won't ship them an intermediate image.

Therefore, it makes sense for us to drop the `-cnb` suffix as it doesn't provide much in the way of information.

Reply:
I'm all for that. Do we think we can cut support of the separate stacks in the Bionic stacks as well?

Reply:
I'd like to see that happen. Maybe we ship the same image to both the `{variant}` and `{variant}-cnb` tags.

Reply:
My hope is that with the addition of the [Stack descriptor RFC](https://github.com/paketo-buildpacks/rfcs/blob/stack-descriptor/text/stacks/0003-stack-descriptor.md) we won't need this type of intermediate image. Folks wanting to recreate our stacks can just use the descriptor and modify it to support their specific use case.

I think we should ship the same image in both locations and then assess the feedback if/when it comes in from users.

Reply:
RE: bionic, is this a note that you could add to the RFC or are you keeping it strictly about Jammy? I'd like our plan documented somewhere.

Reply:
I can add something indicating that we'll only be shipping the one image type, and that there will be no "intermediate" image. The language will be specific to Jammy, and we can choose to adopt that in Bionic as well in some other process or RFC.

Reply:
I've updated the RFC to explain the decision made here.

Reply:
Sounds good to me.